src/components/LineCharts/index.tsx

```diff
@@ -33,17 +37,48 @@ interface IProps {
 echarts.use([LineChart, GridComponent, TitleComponent, TooltipComponent, SVGRenderer]);
 const colors = ['rgba(67, 136, 221, 0.70)', 'rgba(67, 136, 221, 0.30)'];
 
+export const xAxisScalesFunc = (eraPeriod = 86400) => {
+  // if (!currentEra.data?.period) return;
+  const currentDate = dayjs();
+  const intervalPeriod = eraPeriod < 86400 ? 86400 : eraPeriod;
+  const getAxisScales = {
+    l3m: () => {
```

Comment:
[suggestion] Can abstract the calculation for the data source

Reply:
Oh, right. Thanks~, I missed it.